cmd/atlas/inspect.go

```diff
@@ -31,3 +36,16 @@ func init() {
 	)
 	cobra.CheckErr(inspectCmd.MarkFlagRequired("dsn"))
 }
+
+func runInspect(dsn string) {
+	ctx := context.Background()
+	a, err := defaultMux.OpenAtlas(dsn)
+	cobra.CheckErr(err)
+	name, err := schemaNameFromDSN(dsn)
+	cobra.CheckErr(err)
+	schema, err := a.Inspector.InspectSchema(ctx, name, nil)
+	cobra.CheckErr(err)
+	ddl, err := a.MarshalSpec(schema, schemahcl.Marshal)
```

Comment:
see comment on  coupling driver to serialization. 
also the syntax (HCl) will be a parameter in the near future. guessing people will want JSON soon to build tooling on top

Reply:
agree but for now I'll make it default.